sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesResult.cs

```diff
@@ -47,6 +53,12 @@ public partial class AnalyzeHealthcareEntitiesResult : TextAnalyticsResult
         /// </summary>
         public IReadOnlyCollection<TextAnalyticsWarning> Warnings { get; } = new List<TextAnalyticsWarning>();
 
+        /// <summary>
+        /// The language of the input document as detected by the service when requested to perform automatic language
+        /// detection, which is possible by specifying "auto" as the language of the input document.
+        /// </summary>
+        public DetectedLanguage? DetectedLanguage { get; }
```

Comment:
Should this be string for healthcare to match how the service returns it for this api version?

Reply:
Yeah, you're probably right: It might be less confusing if it's simply a string for now. I changed it.